azurelinuxagent/ga/exthandlers.py

```diff
@@ -1515,23 +1530,14 @@ def _truncate_message(field, truncate_size=_MAX_SUBSTATUS_FIELD_LENGTH):
 
 
 class HandlerEnvironment(object):
-    def __init__(self, data):
-        self.data = data
-
-    def get_version(self):
-        return self.data["version"]
 
-    def get_log_dir(self):
-        return self.data["handlerEnvironment"]["logFolder"]
-
-    def get_conf_dir(self):
-        return self.data["handlerEnvironment"]["configFolder"]
-
-    def get_status_dir(self):
-        return self.data["handlerEnvironment"]["statusFolder"]
-
-    def get_heartbeat_file(self):
-        return self.data["handlerEnvironment"]["heartbeatFile"]
+    fileName = "HandlerEnvironment.json"
+    handlerEnvironment = "handlerEnvironment"
+    logFolder = "logFolder"
+    configFolder = "configFolder"
+    statusFolder = "statusFolder"
+    heartbeatFile = "heartbeatFile"
+    eventsFolder = "eventsFolder"
```

Comment:
Is this the value of the folder name or property ? Anyways, name used for the folder in Windows is "Events"

Reply:
This is the name for the property. The name for the folder would be `events` (lower case) in linux too.

Reply:
According to our document - 
>/var/log/azure/ExtensionName/events/ and C:\WindowsAzure\Logs\Plugin\ExtensionName \Events  

Looks like for Linux we use lower-case and for windows we use Capitalised-case. Do you want to keep this or converge the names? I personally dont mind the difference because both of them are specific to the specific OSes (like in linux the standard is to have lower-case names and in windows the standard is to have Capitalised-case.